sdk/identity/azure-identity/azure/identity/__init__.py

```diff
@@ -34,3 +34,6 @@
     "SharedTokenCacheCredential",
     "UsernamePasswordCredential",
 ]
+
+from ._version import VERSION
```

Comment:
That's an identity bug, technically you don't need this for your work, since you have a _version.py :)

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/8664